Packs/Jira/Integrations/JiraEventCollector/JiraEventCollector.yml

```diff
@@ -43,6 +32,18 @@ configuration:
   name: proxy
   required: false
   type: 8
+- defaultvalue: jira
+  additionalinfo: "For the dataset name for example: jira"
+  display: Vendor name
+  name: vendor
+  required: false
+  type: 0
```

Comment:
Need to define a standard here. In other cases, we only gave the product as a param.
Correct me if I am wrong here @yaakovi